opdracht-1.js

```diff
@@ -0,0 +1,44 @@
+console.log("..............1A..............");
+// 1a
+const tvType = inventory.map((tv) => {
+    tv = tv.type
+    return tv
```

Comment:
Dit is een bad practice. Je past een item aan in de array waarover je aan het itereren bent. Als je enkel geïnteresseerd bent in het type, dan kun je die beter meteen teruggeven:
```suggestion
    return tv.type;
```